modules/programs/vscode.nix

```diff
@@ -56,12 +56,29 @@ in {
         '';
       };
 
+      disableUpdateCheck = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Enable to silence update popups. Updates userSettings.
+        '';
+      };
+
+      disableExtensionUpdateCheck = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Enable to silence update notifications for extensions. Updates userSettings.
+        '';
+      };
+
       userSettings = mkOption {
         type = jsonFormat.type;
         default = { };
         example = literalExpression ''
           {
-            "update.channel" = "none";
+            "update.mode" = "none"
+            "extensions.autoCheckUpdates" = false
```

Comment:
```suggestion
```

This doesn't make a whole lot of sense to include here since we are encouraging people to use the other options.

Reply:
Yeah, replaced with a different example for a string arg.